### PR TITLE
mvcc: log when compaction is done

### PR DESCRIPTION
**Pull request by gyuho**
I am debugging `context timeout` issues on compact request call, where I need to look at the time the compact request gets sent out, and the time when the compact operation finishes. (https://github.com/coreos/etcd/pull/5421)

/cc @heyitsanthony @xiang90

### CODE REVIEW

File: mvcc/kvstore_compaction.go [hunk 1]
```diff
@@ -48,6 +49,7 @@ func (s *store) scheduleCompaction(compactMainRev int64, keep map[revision]struc
 			revToBytes(revision{main: compactMainRev}, rbytes)
 			tx.UnsafePut(metaBucketName, finishedCompactKeyName, rbytes)
 			tx.Unlock()
+			log.Printf("mvcc: finished scheduled compaction at %d (took %v)", compactMainRev, time.Since(totalStart))
```
Comment (xiang90) [inline mvcc/kvstore_compaction.go]: we should use plog instead of golang log pkg

**Comment by gyuho** [comment]
@xiang90 Just replaced all `log.` with capnslog under mvcc. PTAL.

Thanks!

**Comment by xiang90** [comment]
lgtm

**Comment by gyuho** [comment]
thanks.

**Pull request merged**